patterns/.github/workflows/lint-test.yaml

```diff
@@ -0,0 +1,54 @@
+name: Lint and Test Charts
```

Comment:
Last thing. I don't think you'll want these here as the actions should be in each individual pattern sub dir

Reply:
> Last thing. I don't think you'll want these here as the actions should be in each individual pattern sub dir

@Chris Sanders I know we discussed yesterday and went with .github/workflows; but I think I like @diamonwiggins idea. What are your thoughts here?

Reply:
What's the suggestion here, `patterns/.github/workflows/lint/lint-test.yaml`?

Reply:
To clarify, i'm not saying that the structure `.github/workflows` should change. I'm saying that inside of the `patterns` directory there will be sub directories for each pattern I imagine and that's where the workflows should live. Not at the root of `patterns/`

Reply:
after slack discussion `patterns/github-workflows/.github/wofklow/<all the yamls>` is the pattern we'll go with for now

Reply:
👍